src/bitmessagekivy/baseclass/payment.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=import-error, no-name-in-module, too-few-public-methods, too-many-ancestors
+
+'''
+    This is for payment related part
+'''
+
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.screenmanager import Screen
+from kivy.app import App
+
+from kivymd.uix.behaviors.elevation import RectangularElevationBehavior
+from kivymd.uix.label import MDLabel
+from kivymd.uix.list import (
+    IRightBodyTouch,
+    OneLineAvatarIconListItem
+)
+
+from bitmessagekivy.baseclass.common import toast
+
+import state
+
+
+class Payment(Screen):
+    """Payment Screen class for kivy Ui"""
+    kivy_running_app = App.get_running_app()
+    kivy_state = kivy_running_app.kivy_state_obj
+
+    def get_free_credits(self, instance):
+        """Get the available credits"""
+        # pylint: disable=no-self-use
+        self.kivy_state.available_credit = instance.parent.children[1].text
+        existing_credits = state.kivyapp.root.ids.sc18.ids.cred.text
```

Comment:
if possible avoid `state`. My guess is `state.kivyapp` is the same as `self.kivy_running_app`